Tests/FluentAssertions.Equivalency.Specs/DataColumnSpecs.cs

```diff
@@ -703,5 +703,27 @@ public void When_ExtendedProperties_do_not_match_and_property_is_excluded_it_sho
             dataColumn1.Should().BeEquivalentTo(dataColumn2,
                 options => options.Excluding(dataColumn => dataColumn.ExtendedProperties));
         }
+
+        [Fact]
+        public void When_object_of_type_unequal_to_DataColumn_is_asserted_with_a_DataColumn_it_fails()
```

Comment:
We're trying to move away from the strict `When_x_it_should_y` naming scheme as the sentences sometimes bend over backwards to exactly fit the pattern.

`Data_column_is_not_equivalent_to_another_type`